Tests/WebSocketKitTests/WebSocketKitTests.swift

```diff
@@ -6,11 +6,18 @@ import NIOHTTP1
 import NIOSSL
 import NIOWebSocket
 @testable import WebSocketKit
+import CompressNIO
 
 final class WebSocketKitTests: XCTestCase {
+    var elg: EventLoopGroup!
+
     override func setUp() async throws {
         fflush(stdout)
     }
+    
+    override func tearDown() {
+        try! self.elg.syncShutdownGracefully()
```

Comment:
Not a fan of force unwrapped optionals. And `try!`. You can make `tearDown()` throwing.

Reply:
I think those were there originally I dont recall writing them.

Reply:
Ive removed the !.s I can find. Not sure what to do about the ones some other maintainer left in there. 
I can try if its important but feel it could unravel things maybe.

Reply:
I dont mind fixing them, but without understanding the test I cant say my change would still be a valid test. I did that in the cases I did change, like with the URL, but didnt want to go down the rabbit hole like a cowboy. Someone should do it I think. Since I already fixed deprecations without asking it's kinda six of one half dozen of another.